src/Symfony/Component/Lock/Tests/LockFactoryTest.php

```diff
@@ -15,7 +15,7 @@
 use Psr\Log\LoggerInterface;
 use Symfony\Component\Lock\LockFactory;
 use Symfony\Component\Lock\LockInterface;
-use Symfony\Component\Lock\StoreInterface;
+use Symfony\Component\Lock\PersistingStoreInterface;
```

Comment:
for 4.4

Reply:
This one is not possible in 4.4 because the factory still implements type-hint StoreInterface, for the other one see https://github.com/symfony/symfony/pull/32678